src/chains/models.py

```diff
@@ -119,6 +119,7 @@ class RpcAuthentication(models.TextChoices):
     recommended_master_copy_version = models.CharField(
         max_length=255, validators=[sem_ver_validator]
     )
+    warning = models.CharField(max_length=255, blank=True, null=True)
```

Comment:
Maybe a `TextField`?

Reply:
Yes, thanks! This is something I forgot to annotate in the PR, but this made me doubt as well 👍

Reply:
Changed in [02e17b2](https://github.com/safe-global/safe-config-service/pull/688/commits/02e17b2aadef9298675f6238cfaf769509feb33c)